src/main/java/org/recordrobotics/charger/commands/auto/AutoMoveArm.java

```diff
@@ -86,38 +95,10 @@ public void execute() {
 				angles = _placehold;//_arm.getAnglesOfRotation(_flipGroundChangeX, _flipGroundChangeY);
 				break;
 			default:
-				angles = _arm.resetPositions();
+				angles = _neutral;
 				break;
 		}
-		_originPid.setSetpoint(angles[0]);
-		_changePid.setSetpoint(angles[1]);
-		double _originSpeed = _originPid.calculate(_arm.getOriginEncoder());
-		double _changeSpeed = _changePid.calculate(_arm.getChangeEncoder());
-		if(Math.abs(_originSpeed) > _maxSpeed){
-			if(Math.sin(Units.degreesToRadians(_arm.getOriginEncoder())) < -Math.sqrt(2)/2){
-				_originSpeed = _maxDownSpeed * Math.signum(_originSpeed);
-			}else{
-				_originSpeed = _maxSpeed * Math.signum(_originSpeed);
-			}
-		}
-		if(Math.abs(_changeSpeed) > _maxSpeed){
-			if(Math.sin(Units.degreesToRadians(_arm.getChangeEncoder() + _arm.getOriginEncoder() * 5/7)) < -Math.sqrt(2)/2){
-			_changeSpeed = _maxDownSpeed * Math.signum(_changeSpeed);
-			}else{
-				_changeSpeed = _maxSpeed * Math.signum(_changeSpeed);
-			}
-		}
-
-		_arm.spinOrigin(_originSpeed);
-		_arm.spinChange(_changeSpeed);
-	}
-
-	public boolean originIsFliped(){
-		return _originPid.atSetpoint();
-	}
-
-	public boolean _changeIsFliped(){
-		return _changePid.atSetpoint();
+		_arm.setAngles(angles);
```

Comment:
I think it's way better to do something like `_arm.setPose(THIRD)` .. and don't let people set angles on the arm. Let the arm only know about poses and that's the main vocabulary to control it. Make it consistent. -- this means also that arm does not expose its current angles, so for `isFinished()`, you'd call `_arm.atPose(THIRD)` or something like that